pkg/virt-controller/services/template_test.go

```diff
@@ -2573,6 +2573,55 @@ var _ = Describe("Template", func() {
 
 		})
 
+		Context("Ephemeral storage request", func() {
+
+			table.DescribeTable("by verifying that ephemeral storage ", func(defineEphemeralStorageLimit bool) {
+				vmi := v1.NewMinimalVMI("fake-vmi")
+
+				ephemeralStorageRequests := resource.MustParse("30M")
+				ephemeralStorageLimit := resource.MustParse("70M")
+				ephemeralStorageAddition := resource.MustParse("50M")
+
+				if defineEphemeralStorageLimit {
+					vmi.Spec.Domain.Resources = v1.ResourceRequirements{
```

Comment:
Sorry but I can't see how does the VMI resource request affects the container disk resource request, aren't the container disk resource request is hard-coded inside the `GenerateContainers` function?

Reply:
As I wrote to vladik above: the thing is that if the user requests X MBs of ephemeral-storage then the amount that should be requested is X+50Mb since we should guarantee the user can use all of his requested storage.

Reply:
@enp0s3 we need to introduce this request to both containers.